pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -468,6 +468,19 @@ def test_mismatched_length_cmp_op(cons):
         left & right
 
 
+@pytest.mark.parametrize(
+    "a, b",
+    [
+        (SparseArray([0, 1, 2], fill_value=0), np.array([0, 1, 2, 3])),
```

Comment:
Nit: Could you move the `SparseArray` and `np.array` call into the test body?

Reply:
Done